### DISCUSSION TITLE
is there library for mesh analysis with jax support?

### DISCUSSION THREAD

**Discussion by jakubMitura14** [jax-ml/jax#12034] [Q&A] [answered]
Hello I am developing some model, where part will be CNN part graph neural network and part based on mesh analysis - and now my question is is there some library that support mesh data structure sth like https://pytorch3d.org/ and will play well with jax (that I could run backpropagation fully through the whole model)

**Comment by adam-hartshorne** [comment, marked as answer]
https://github.com/ml-for-gp/jaxgptoolbox

**Reply by jakubMitura14** [reply]
thanks!
